packages/schemas/src/foundations/jsonb-types/sign-in-experience.ts

```diff
@@ -105,8 +105,12 @@ export const mfaFactorsGuard = z.nativeEnum(MfaFactor).array();
 export type MfaFactors = z.infer<typeof mfaFactorsGuard>;
 
 export enum MfaPolicy {
```

Comment:
Need to add some comments to explain the definition and use case of these values.